### PR TITLE
Fix Issue 20025 - alias this combined with a copy constructor seems to lead to undefined behaviour.

### PR DESCRIPTION
**Pull request by RazvanN7**

### CODE REVIEW

**Review by thewilsonator — APPROVED** [review]

**Comment by Basile-z** [comment]
LGTM but can I suggest to target stable now that 2.087 is released ?

**Comment by RazvanN7** [comment]
@Basile-z done.

**Review by ghost — APPROVED** [review]
OK

File: test/runnable/test20025.d [hunk 1]
```diff
@@ -0,0 +1,27 @@
+// https://issues.dlang.org/show_bug.cgi?id=20025
+/*
+TEST_OUTPUT:
+---
+---
+*/
+
+struct B
+{
+        static int value = 77;
```
Comment (JinShil) [inline test/runnable/test20025.d]: Indentation looks too deep.

**Pull request merged**